src/library_glfw.js

```diff
@@ -1337,7 +1407,10 @@ var LibraryGLFW = {
   },
 
   glfwWindowHint: function(target, hint) {
-    GLFW.hints[target] = hint;
+    if(GLFW.hints === null)
```

Comment:
Space after `if` and curly braces around the if/else arms.

Also, this is seems like maybe another separate change unrelated dto multi-window?

Reply:
You are right. thx! anyway I haven't commited the most recent version yet because I have to rebase it.